app/views/devise/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if resource.errors.any? %>
+  <div id="error_explanation">
+    <h2>
+      <%= I18n.t("errors.messages.not_saved",
+                 count: resource.errors.count,
+                 resource: resource.class.model_name.human.downcase)
+       %>
+    </h2>
+    <ul>
+      <% resource.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+    </ul>
+  </div>
+<% end %>
```

Comment:
- [ OPTIONAL ] I would highly recommend to add some styles to your error message to make your project has more human sense and that's important for user experience ✅✨💯🚀

Reply:
Done